adapter-mongodb/src/main/resources/activities/mongodb-crud-basic.yaml

```diff
@@ -7,11 +7,11 @@ description: |
 
 scenarios:
   default:
-    schema:   run driver=mongodb tags==phase:schema threads==1 cycles==UNDEF
-    write:    run driver=mongodb tags==phase:main,type:write cycles===TEMPLATE(write-cycles,TEMPLATE(docscount,10000000)) threads=auto errors=timer,warn
-    read:     run driver=mongodb tags==phase:main,type:read cycles===TEMPLATE(read-cycles,TEMPLATE(docscount,10000000)) threads=auto errors=timer,warn
-    update:   run driver=mongodb tags==phase:main,type:update cycles===TEMPLATE(update-cycles,TEMPLATE(docscount,10000000)) threads=auto errors=timer,warn
-    delete:   run driver=mongodb tags==phase:main,type:delete cycles===TEMPLATE(delete-cycles,TEMPLATE(docscount,10000000)) threads=auto errors=timer,warn
+    schema:   run driver=mongodb tags==block:\"schema.*\" threads==1 cycles==UNDEF
```

Comment:
Can we use `'` instead of `\"` for these? Like `tags==block:'schema.*'` or `tags='block:schema.*'` ?